funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -617,8 +418,9 @@ def _main_loop(self):
 
                 # results will be a pickled dict with task_id, container_id,
                 # and results/exception
-                self.results_outgoing.put("forwarder", results["message"])
-
+                log.warning(f"Publishing message {results['message']}")
```

Comment:
```suggestion
                log.info(f"Publishing message {results['message']}")
```

Reply:
Oh boy, we need to remove this line instead.